# A Case Study {#cs_RNA}

Lets take a simple RNA seq experiment and follow the steps from start to finish.

### Set up the project folder and repo {-#csrna_setup}

First, I navigate to my projects folder. Now, since this will be an RNA seq project, I clone the RNA seq template from github:
```{sh eval=F}
git clone --depth 1 https://github.com/NielInfante/RNAseq.git Case_study_2019_02
```

This will create the folder Case_study_2019_02, and copy everything from the template repository into it. The depth flag tells git that you only want the latest code, without all the revision history, to save a little space and time.

Now go to github and create a repository called Case_study_2019_02, and use the option to create a repo from an existing repository. Do not initialize with a README.

Then, using your github ID and the name you want for the project:
```{sh eval=F}
git remote add origin https://github.com/NielInfante/Case_study_2019_02.git
git push -u origin master
```

You now have your project inititalized and connected to a github repository.


### Get the data {-#csrna_getData}

Should I do this with Agazie data?  Project ID 22934923

I will be getting the data from BaseSpace. See \@ref(basespace). The project ID is 22934923. This is a Human RNA seq experiment, explaination here **Link to paper, if available**.



Its Human RNA, its old. Check with him, it may be published by now.

Use getFastq.R to get data

### Create Conda Environment {-csrna_conda}

Find the versions that you want. You should install specific versions (generally the latest), and keep track of the version for reproducibility purposes. Copying the conda create command to your readme will record this.

```{sh eval=F}
conda search fastqc
conda search multiqc
conda search salmon

conda create -n case_study_2019_02 fastqc=0.11.8 multiqc=1.7 salmon=0.12.0




```
### Do QC {-#csrna_doQC}

fastqc and multiqc

### Quantification {-#csrna_quant}

Get transcripts, merge them, get IDs

conda salmon
  build index
  do quantification

### Differential Expression {-#csrna_diff}

DESeq2


### GO, GSEA, pathway {-#csrna_go}
